docs/useNotify.md

```diff
@@ -31,8 +31,10 @@ The callback takes 2 arguments:
 Here are more examples of `useNotify` calls: 
 
 ```js
+// notify an error
+notify(`This is an error`, { type: 'error' });
 // notify a warning
-notify(`This is a warning`, { type: 'warning' });
+notify(`This is a warning`, { type: 'error' });
```

Comment:
this change is incorrect

Reply:
Indeed

It's back to `warning` : 6be244017c22f9eb51f228d5ce4636352635be55